### PR TITLE
Improve SubProject documentation

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,11 @@ The following CMake modules can be included in your project:
       ENABLE_CLANGCHECK_TESTS is set.
 * [PackageConfig](PackageConfig.cmake) generates cmake package information files
   for the project. These files let other CMake-based projects locate it through
-  find_package (in config mode, without the need for a finder script).
+  find_package (in config mode, without the need for a finder script). Must be
+  included at the end of the CMakeLists.txt, after all targets have been added.
+* [CommonCPack](CommonCPack.cmake) Configures the CPack pacakge generator to
```
Comment (eile) [inline README.md]: Typo pacakge

**Comment by eile** [comment]
+1 otherwise - merge at will.

File: PackageConfig.cmake [hunk 2]
```diff
@@ -6,6 +6,14 @@
 # files. Those files are used in the config-mode of find_package which
 # supersedes the Find${PROJECT_NAME}.cmake file.
 #
+# Two sets of the abovementioned files are generated:
+# * One in the install directory, so that the project can be found once it has
+#   been depoloyed.
```
Comment (tribal-tec) [inline PackageConfig.cmake]: typo depoloyed

Comment (rdumusc) [inline PackageConfig.cmake]: Ouch, already merged.. :-( will try to remember to fix it in a next commit...

**Pull request merged**